Dividir_arr_1/d_array1.js

```diff
@@ -0,0 +1,64 @@
+alert("Programa que divide arrays por el tipo de valor");
+
+let array_general = []; //Almacén general del input
+
+let i = 0;
+let input_grl = prompt("Ingresa los números que desees. \nCuando finalices escribe el número cero.");
+while (input_grl != "0") { //Mientras el input sea diferente de cero continúa...
+    array_general[i] = input_grl; //Guardar los valores de input_grl en un array_general    
+    input_grl = prompt("Ingresa los números que desees. \nCuando finalices escribe el número cero.");
+    i++;
+}
+
+console.log(array_general);
+
+function n_pares(array) {
+    let almacen_pares = [];    
+    for (i = 0; i < array.length; i++) {
```

Comment:
Recuerda utilizar nombres descriptivos para las variables, en lugar de **i** puedes usar **index**. 
La instrucción `i < array.length` no tiene un buen rendimiento para arreglos con muchos elementos, es mejor asignar el tamaño del arreglo a una variable ya sea en el mismo for o fuera del for y realizar la comparación con esta variable. 
La instrucción `i++` no tiene un buen rendimiento en su lugar es mejor utilizar `i += 1`
La cabecera del for puede quedar de la siguiente forma:

>  `for (let index = 0, arrayLength = array.length; index < arrayLength; index += 1)`

Reply:
ok cambiado!